Automattic/wp-calypso
# Search Exp: Search Text-Only search on LoTS

Related to https://github.com/Automattic/dotcom-forge/issues/2409

## Proposed Changes

* Split the feature flag for LoTS and LiTS.
* Created a new search component that doesn't have the filter dropdown. I've leveraged the TS architecture to reduce the number of props we use in this new component.

## Testing Instructions



* Open the Calypso Live Link that you can find in this PR
* Navigate to the LoTS, and click on the search bar. You shouldn't see the dropdown menu with filters.
* If you add filters to the URL the added filters should continue to work.
* Navigate to the LiTS, and use the search bar. You shouldn't see the dropdown menu with filters.
* If you add filters to the URL the added filters should continue to work.


|Before|After|
|-----|-----|
|![Captura de pantalla 2023-09-28 a las 14 18 29](https://github.com/Automattic/wp-calypso/assets/1989914/6c11f6c2-5b00-43f3-8eeb-1eb63d4ecb73)|![Captura de pantalla 2023-09-28 a las 14 18 09](https://github.com/Automattic/wp-calypso/assets/1989914/f84c6628-9305-4906-997c-a751cd56a174)|


## Pre-merge Checklist
